data/fro/Vietor_1876_Loherains/stemma.gv

```diff
@@ -0,0 +1,62 @@
+digraph {
+alpha1[label="α1", color="grey"];
+alpha[label="α", color="grey"];
+beta[label="β", color="grey"];
+chi[label="χ", color="grey"];
+delta[label="δ", color="grey"];
+epsilon[label="ε", color="grey"];
+eta[label="η", color="grey"];
+gamma[label="γ", color="grey"];
+ksi[label="ξ", color="grey"];
+lambda[label="λ", color="grey"];
+mu[label="μ", color="grey"];
+nu[label="ν", color="grey"];
+omicron[label="ο", color="grey"];
+t[color="grey"];
+theta[label="θ", color="grey"];
+zeta[label="ζ", color="grey"];
+alpha1->alpha;
+alpha->B;
+alpha->A;
+alpha->K;
+alpha->J;
+alpha->beta;
+beta->C;
+beta->gamma;
+gamma->O;
+gamma->R;
+gamma->delta;
+delta->epsilon;
+epsilon->G;
+epsilon->D;
+delta->zeta;
+zeta->F;
+zeta->eta;
+eta->a;
+eta->S;
+eta->Q;
+eta->Z;
+eta->theta;
+theta->T;
+theta->t;
```

Comment:
I think "t" is supposed to be "ι" (iota). It follows "θ" in the Greek alphabet.

Reply:
Thanks, I need to revise my Greek alphabet.